cdk/v2/pinterest_tag/rtWorkflow.yaml

```diff
@@ -0,0 +1,70 @@
+bindings:
+  - path: ../../../v0/destinations/pinterest_tag/config
+steps:
+  - name: prepareContext
+    template: $setContext("batchMode", true)
+  - name: transform
+    externalWorkflow:
+      path: ./procWorkflow.yaml
+    loopOverInput: true
+  - name: processTransformOutput
+    description: Separate Failed and successful events
+    template: |
+      (
+        $zipInputAndOutput := $map($zip($, $outputs.transform), $merge);
```

Comment:
Read documentation of $zip but still unclear about what is happening on this line

Reply:
https://try.jsonata.org/V-hStnHKA play around with this and let me know if still need clarification